### PR TITLE
[MERGEABLE] Various issues

### PR DESCRIPTION
**Pull request by willyaranda**
This fixes issues #68, #74, #75, #80 and more bugs found with the test cases ;)

### CODE REVIEW

**Comment by willyaranda** [comment]
There is a problem with the certificate. Let's check tomorrow

**Comment by frsela** [comment]
Which kind of problem?

**Comment by willyaranda** [comment]
Node is decoding correctly the PbK from the database (stored in base64), but there are some kind of problem with openssl, which is generating an error. I'll post here in a few minutes.

File: other/post-to-websocket.js [hunk 1]
```diff
@@ -9,62 +9,95 @@ function onHTTPMessage(req, res) {
...
-  client.on('connectFailed', function(error) {
-    console.log('Connect Error: ' + error.toString());
-  });
+function websocket(uatoken, text, callback) {
+  if (!connectionsText[uatoken]) connectionsText[uatoken] = '';
+  var messageType = '';
+  try {
+    messageType = JSON.parse(text).messageType;
+  } catch(e) {
+    console.log('no se puede parsear respuesta de websocket');
```
Comment (frsela) [inline other/post-to-websocket.js:53]: NIT: Please, in english ;p

File: test/functions/E2E.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ var debug = require('./common').debug;
   getToken: function getToken() {
     PushTest.port =  require('../../src/config.js').NS_UA_WS.interfaces[0].port;
     PushTest.host = '127.0.0.1';
-    PushTest.NOTIFICATION = '{"messageType":"notification","id":1234,"message":"Hola","signature":"","ttl":0,"timestamp":"SINCE_EPOCH_TIME","priority":1}';
+    PushTest.NOTIFICATION = '{"messageType":"notification","id":1234,"message":"Hola","signature":"691cb72015afdba8742349431500b497fe689523c7bd8b9ab9d905160efed20e8c70e7ba1aec112c494721f253b8874f90d611b8ebd78e5017aaf971f0f01503e2d3ba1949cd11c145f0537b7c80a7933368f405d12b723f8107c92af1e1d58a93c48a9af3f55ee519719b8ba1632e1fd12f9d3eb99846abb849793516bf1fa0","ttl":0,"timestamp":"SINCE_EPOCH_TIME","priority":1}';
```
Comment (frsela) [inline test/functions/E2E.js:28]: Nit: Instead SINCE_EPOCH_TIME use a real timestamp like in

https://github.com/telefonicaid/notification_server/blob/develop/test/scripts/send_notification.sh

in order to be used in a near future ;)

**Pull request merged**